server/pkg/tfchain/client.go

```diff
@@ -642,3 +655,34 @@ func (c *Client) SwapToStellar(ctx context.Context, conState jsonrpc.State, args
 
 	return state.client.SwapToStellar(state.identity, args.TargetStellarAddress, *args.Amount)
 }
+
+func (c *Client) AwaitTransactionOnTfchainBridge(ctx context.Context, conState jsonrpc.State, memo string) error {
+	state := State(conState)
+	if state.client == nil {
+		return pkg.ErrClientNotConnected{}
+	}
+	_, err := tfchainBridgeAddressFromNetwork(state.network)
+	if err != nil {
+		return err
+	}
+
+	for i := 0; i < int(300); i++ {
```

Comment:
make this a `const`

Reply:
> While the code looks great, I think the example should not ask for a Stellar and Tfchain key.
> 
> The main reason we want to have this example is to have a seamless experience for an ethereum user having only an ethereum key. All the other keys should be generated on the fly as they are not provided by the user.

It makes sense to create the account for Stellar but not really the account on Tfchain. Because the user will have to cancel the deployment later, right? Or he might want to get information on the deployment later and for that he needs the tfchain account.

Reply:
The user will need to extract the key (some method needs to exist to export a tfchain private key). The idea is that the user only has an ethereum key, if we introduce the concept that he also needs to tfchain key we will make it harder to complete this flow.